game.js

```diff
@@ -57,28 +54,23 @@ class Game {
       <img src="./assets/${classicComputerChoice}.png" alt="${classicComputerChoice}">
       `
     } else {
-
+      //Determine Difficult Winner
       var difficultComputerChoice = this.randomChoice();
       var difficultUserChoice = choice;
 
-      if ( (difficultComputerChoice === "rock" && difficultUserChoice === "rock")
-        || (difficultComputerChoice === "paper" && difficultUserChoice === "paper")
-        || (difficultComputerChoice === "scissors" && difficultUserChoice === "scissors")
-        || (difficultComputerChoice === "lizard" && difficultUserChoice === "lizard")
-        || (difficultComputerChoice === "alien" && difficultUserChoice === "alien") ) {
-
+      if (difficultComputerChoice === difficultUserChoice) {
         this.tieAlert();
 
-      } else if ( (difficultComputerChoice === "rock" && difficultUserChoice === "lizard")
-        || (difficultComputerChoice === "rock" && difficultUserChoice === "scissors")
-        || (difficultComputerChoice === "paper" && difficultUserChoice === "scissors")
-        || (difficultComputerChoice === "paper" && difficultUserChoice === "alien")
-        || (difficultComputerChoice === "scissors" && difficultUserChoice === "paper")
-        || (difficultComputerChoice === "scissors" && difficultUserChoice === "lizard")
-        || (difficultComputerChoice === "lizard" && difficultUserChoice === "paper")
-        || (difficultComputerChoice === "lizard" && difficultUserChoice === "alien")
-        || (difficultComputerChoice === "alien" && difficultUserChoice === "rock")
-        || (difficultComputerChoice === "alien" && difficultUserChoice === "scissors") ) {
+      } else if ((difficultComputerChoice === "rock" && difficultUserChoice === "lizard") ||
```

Comment:
Whew! That's one giant else if statement lol

I bet there's a way to dry this up a bit. A helper function or two might be a good call for this one.